scripts/yocto-build-and-test.sh

```diff
@@ -204,12 +204,22 @@ PREFERRED_VERSION${sep}pn-mender-configure = "$MENDER_CONFIGURE_MODULE_VERSION"
 EOF
     fi
 
+    if [ "${MENDER_REV}" = "feature-c++-client" ]; then
```

Comment:
This won't work for PRs :thinking:

Reply:
Hmm, good point. We have no good way of getting this info do we?

Initially I had this PR: https://github.com/mendersoftware/integration-test-runner/pull/195

Better approach you think?

Reply:
> Hmm, good point. We have no good way of getting this info do we?

Not from mender-qa only, no.

> Initially I had this PR: [mendersoftware/integration-test-runner#195](https://github.com/mendersoftware/integration-test-runner/pull/195)
> 
> Better approach you think?

I am afraid we need something like that: a way from the build trigger-er to explicitly ask for these tweaks in `mender-qa`.

Reply:
Decided to go with the variable for now

Reply:
The `BUILD_CPP_CLIENT` variable from the `integration_test_bot` that is